web/src/views/DandisetLandingView/DandisetDetails.vue

```diff
@@ -0,0 +1,168 @@
+        :class="rowClasses"
+      >
+        <v-icon>mdi-calendar</v-icon>
+        <span :class="labelClasses">Created on</span>
+        <span :class="itemClasses">{{ created }}</span>
+      </v-row>
+      <v-row
+        no-gutters
+        :class="rowClasses"
+      >
+        <v-icon>mdi-update</v-icon>
+        <span :class="labelClasses">Last updated</span>
+        <span :class="itemClasses">{{ lastUpdated }}</span>
+      </v-row>
+
+      <v-divider class="my-2" />
+
+      <v-row :class="`${rowClasses} px-2`">
+        <span :class="labelClasses">Identifier</span>
+        <span :class="itemClasses">{{ currentDandiset.meta.dandiset.identifier }}</span>
+      </v-row>
+
+      <v-divider class="my-2 px-0 mx-0" />
+
+      <v-row>
+        <v-col>
+          <v-card
+            color="grey lighten-3"
+            class="mx-2"
+            flat
+            tile
+          >
+            <v-row
+              no-gutters
+              class="py-2"
+            >
+              <v-col cols="11">
+                <v-icon
+                  class="mx-2"
+                  color="primary"
+                >
+                  mdi-link
+                </v-icon>
+                <span :class="itemClasses">Draft</span>
+              </v-col>
+              <v-col>
+                <v-tooltip top>
+                  <template v-slot:activator="{ on }">
+                    <v-icon
+                      small
+                      color="grey darken-1"
+                      v-on="on"
+                    >
+                      mdi-help-circle
+                    </v-icon>
+                  </template>
+                  <span>This is a version of your dandiset that contains unpublished changes.</span>
+                </v-tooltip>
+              </v-col>
+            </v-row>
+          </v-card>
+        </v-col>
+      </v-row>
+      <!-- TODO: Uncomment this once the versions API is accessible -->
+      <!-- <v-row>
+        <v-col class="pa-0">
+          <v-card
+            color="grey lighten-2"
+            tile
+            flat
+          >
+            <div class="py-1 px-3">
+              Versions
+            </div>
+          </v-card>
+        </v-col>
+      </v-row>
+
+      <v-row>
+        <v-timeline>
+          <v-timeline-item
+            small
+            right
+          >
+            <template v-slot:opposite>
+              <span class="caption text--secondary">
+                02/26/19
+              </span>
+            </template>
+
+            <span class="font-weight-medium">
+              1.2.1
+            </span>
+          </v-timeline-item>
+        </v-timeline>
+      </v-row> -->
+    </template>
+  </v-card>
+</template>
+
+<script>
+import { mapState } from 'vuex';
+import moment from 'moment';
+
+
+export default {
+  name: 'DandisetDetails',
+  data() {
+    return {
+      rowClasses: 'my-1',
+      labelClasses: 'mx-2 text--secondary',
+      itemClasses: 'font-weight-medium',
```

Comment:
These work and are fine, but they seem just a tad smelly to me. Having presentational code like this in the component logic feels like violating some separation of concerns. I do see the benefit of not repeating these classes in every row definition and how that makes it easier to change later.
The only alternative I could think of to keep this tidy would be to make another component specifically for data rows. Do you think that's overkill?
To be clear I'm willing to accept this, I'm just thinking of alternatives.

Reply:
I think having a separate component just for this is a bit overkill, I see your point though. Would scoping these fields under a `style` field make it a bit cleaner?

Reply:
Not really, but if it makes you feel better go for it ¯\_(ツ)_/¯

Reply:
Alright, well since it doesn't look like we have a solution to this right now, I think we can merge this and deal with that later. Also you dropped this  \